src/app/core/common-components/entity-form/entity-form/entity-form.component.ts

```diff
@@ -36,13 +36,7 @@ import { FormsModule } from "@angular/forms";
   // Use no encapsulation because we want to change the value of children (the mat-form-fields that are
   // dynamically created)
   encapsulation: ViewEncapsulation.None,
-  imports: [
-    FormsModule, // importing FormsModule ensures that buttons anywhere inside do not trigger form submission / page reload
```

Comment:
@sadaf895 don't remove code comments, it may be helpful? (FormsModule, // importing FormsModule ensures that buttons anywhere inside do not trigger form submission / page reload)